source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj

```diff
@@ -2279,6 +2279,10 @@
 					"DEBUG=1",
 					"$(inherited)",
 				);
+				"GCC_PREPROCESSOR_DEFINITIONS[arch=*]" = (
```

Comment:
if your intention is to define it for all the flavors, what is the need for **[arch=*]**?
Can we simply:
`GCC_PREPROCESSOR_DEFINITIONS = (
                "DEBUG=1",
                "USEFLUENTUI=1",
                "$(inherited)",
            );`

Reply:
good catch